### ISSUE TITLE
clarification on additional `motivations` for Search API

### ISSUE THREAD

**Issue by mixterj** [IIIF/iiif.io#2053]
The Search API defines a set of possible Motivations (https://preview.iiif.io/api/search-v3-alignment/api/search/2.0/#query-parameters) as well as defines a pattern to using additional motivations from the Web Annotation Ontology. The definition of using other motivations is not clear and should be reviewed for current Search 2.0 Alignment work

**Comment by azaroth42** [comment]
Previous discussion about changing either params (include / exclude) or the value to allow +/- in the value.

Should just reference the registry of motivations.

**Comment by glenrobson** [comment]
Existing motivation registry (which is empty):

https://iiif.io/api/registry/motivations/

**Comment by tomcrane** [comment]
Discussed on Search TSG call 2021-10-19

**Comment by digitaldogsbody** [comment]
Add new contextualizing motivation to the registry

**Comment by kirschbombe** [comment]
I have a PR for updating the motivations registry here:

**Comment by kirschbombe** [comment]
The motivations registry will be merged shortly. I'll update the v3 branch to just reference the updated registry. We'll still need to add the links for the `highlighting` and `contextualizing` motivations to the registry once the v3 alignment branch is merged. I'll create a new ticket for that work.

**Comment by kirschbombe** [comment]
I created a new ticket for @azaroth42's comment about changing either params (include / exclude) or the value to allow +/- in the value:  https://github.com/IIIF/api/issues/2134
